modules/imaging_browser/test/TestPlan.md

```diff
@@ -41,3 +41,8 @@ prepended with the Patient Name.
 25. On the view session page, click the button "Visit Level Feedback".
 26. The entries in this dialog should be editable when a user has the permission `imaging_browser_qc`.
 27. Try editing comments (adding new ones, deleting old ones). Click Save and ensure the data is saved.
+
+### Test the Candidate Dashboard widget
+28. Go to the candidate dashboard in the candidate module and check the Imaging QC Summary widget.
```

Comment:
It seems to be missing tests for the user functionality that is described in the widget itself (ie hovering, clicking..)